WalletWasabi/WabiSabi/Client/CredentialDependencies/CredentialDependency.cs

```diff
@@ -0,0 +1,19 @@
+namespace WalletWasabi.WabiSabi.Client.CredentialDependencies
+{
+	// make private inner class of Graph class?
```

Comment:
That is fine, less confusion in the big class. Usually, there is a folder in the project if there are more objects belong under one functionality.